src/client/Microsoft.Identity.Client/Http/HttpManagerManagedIdentity.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.Core;
+
+namespace Microsoft.Identity.Client.Http
+{
+    /// <remarks>
+    /// Http Manager specific to managed identity to implement the retry for specific http status codes.
+    /// </remarks>
+    internal class HttpManagerManagedIdentity : HttpManagerWithRetry
+    {
+        public HttpManagerManagedIdentity(IMsalHttpClientFactory httpClientFactory) : 
+            base(httpClientFactory) { }
+
+        /// <summary>
+        /// Retry policy specific to managed identity flow.
+        /// Avoid changing this, as it's breaking change.
+        /// </summary>
+        protected override bool IsRetriableStatusCode(int statusCode)
```

Comment:
nit:  `Retryable` 

because https://english.stackexchange.com/questions/305273/retriable-or-retryable and because public API `IsRetryable`